backend/app/views/spree/admin/taxons/_taxon_table.html.erb

```diff
@@ -8,12 +8,12 @@
       </tr>
     </thead>
     <tbody>
-      <% taxons.each do |taxon| %>
+      <% taxons.each do |taxonp| %>
         <tr id="<%= spree_dom_id(taxon) %>" data-hook="taxons_row">
           <td><%= taxon.name %></td>
           <td><%= taxon_path taxon %></td>
           <td class="actions">
-            <%= link_to_delete taxon, url: remove_admin_product_taxon_url(@product, taxon), name: icon('delete.svg) + ' ' + Spree.t(:remove) %>
```

Comment:
This here seems to be missing a closing `'` I could not find where this view file is being displayed to check the fix, so I simplified it to just the word `remove`.